config/develop/glue-job-JSONToParquet.yaml

```diff
@@ -1,15 +1,13 @@
 template:
   path: glue-job-JSONToParquet.j2
 dependencies:
-  - develop/s3-processed-data-bucket.yaml
```

Comment:
So, did this exist because this job would fail without these buckets?

Reply:
Hmmm my thought process behind this is generally we'd want to create all of our buckets and leave them alone prior to running our pipelines I believe so they would all exist.

The removal of the buckets as dependencies and instead relying on referencing it through the `{{ stack_group_config.input_bucket_name }}` is to just reduce redundancy because a lot of the stacks use the buckets as dep. One example is that `cfn-s3objects-macro` won’t run every time we, for example, update the lambda stack because the input bucket is not a dependency for the lambda stack anymore. And if there are any updates to the buckets themselves they would be updated anyways with our CI/CD..

Reply:
We don't need the bucket stack to deploy this resource. If the buckets didn't exist - yes - the `TempS3Bucket` and `S3ScriptBucket` parameters would point to a non-existent location and the job would fail if we tried to run it. But there's nothing preventing us from creating the job before the buckets come into existence.